client/src/components/ProductOverview/ProductOverview.jsx

```diff
@@ -4,24 +4,27 @@ import ImageGallery from './ImageGallery';
 import ProductInfo from './ProductInfo';
 import StyleSelector from './StyleSelector';
 import AddToCart from './AddToCart';
+import { getAverageStars, getStarsByQuarter } from '../../lib/starRatings';
 import s from './style.css';
 import axios from 'axios';
 
 const ProductOverview = () => {
-  const [styleInfo, setStyleInfo] = useState([]);
+  const [data, setData] = useState([]);
   const [productInfo, setProductInfo] = useState({});
-  const id = 65633;
-  // const id = Math.floor((Math.random() * 1011) + 65631)
+  const id = Math.floor((Math.random() * 1011) + 65631);
```

Comment:
for id, do we want to get the random item when we first enter webpage? or are we gonna let users type specific item on endpoint? 
if former, we can use this, but if latter, we should use useParam or querystring I think

Reply:
or if it's temporary one, we can fix it later :)

Reply:
Yes, random product id is temporary, will change it to use Params later